### DISCUSSION TITLE
ASReview label missing records

### DISCUSSION THREAD

**Discussion by goodjobvicky1** [asreview/asreview#1746] [Q&A]
Someone is using ASRview to screen paper for a systematic review. However, when the guy reopened ASRview, his screening records were gone. Is this possible to happen since ASReview is using active learning model that all records should have been recorded while screening.

**Comment by J535D165** [comment]
Thanks for reporting. Was this person using a local installation of a review or hosted version (on a web domain)?

**Reply by goodjobvicky1** [reply]
We use local installation.

**Reply by J535D165** [reply]
hmmm. Can you export the Project and send it to asreview@uu.nl? It might help to figure out what happened.

**Reply by goodjobvicky1** [reply]
Thanks for helping. I have sent you an email with the project.
